src/stravalib/model.py

```diff
@@ -281,7 +283,8 @@ def check_valid_latlng(cls, values: list[float]) -> Optional[list[float]]:
 
         """
         # Strava sometimes returns empty list in case of activities without GPS
-        return values if values else None
+        a = values if values else None
```

Comment:
this was just me debugging.. i'll remove once i understand what is going on here.